Remove `limit` value clamping

parfeon opened the pull request:
fix(heartbeat): fix `state` query parameter

Send proper object for `state` with the `heartbeat` request.

refactor(here-now): remove `limit` value clamping

Don't cap `limit` and let the server perform value validation.

refactor(subscribe): replace legacy `t` with `tt` and `tr`

Replace legacy timetoken query parameter with subscribe v2 parameters.

parfeon committed:
refactor(here-now): remove `limit` value clamping

Don't cap `limit` and let the server perform value validation.

refactor(subscribe): replace legacy `t` with `tt` and `tr`

Replace legacy timetoken query parameter with subscribe v2 parameters.

fix(heartbeat): fix `state` query parameter

Send proper object for `state` with the `heartbeat` request.

pubnub-ops-terraform commented:
### :white_check_mark: **Snyk checks have passed. No issues have been found so far.**

| Status | Scanner | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-critical.svg) Critical | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg) High | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg) Medium | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-low.svg) Low | Total (0) |
|---|---|---|---|---|---|---|
| :white_check_mark: | Licenses| 0 | 0 | 0 | 0 | [ 0 issues](https://app.snyk.io/org/sdk/pr-checks/d5a934ed-2e19-40e3-89b3-d9feea47bd0d/license?refs=b705696fc879cd53980cd150e1743ef0c56863e3&source=prChecksComment) |
| :white_check_mark: | Open Source Security| 0 | 0 | 0 | 0 | [ 0 issues](https://app.snyk.io/org/sdk/pr-checks/d5a934ed-2e19-40e3-89b3-d9feea47bd0d?refs=b705696fc879cd53980cd150e1743ef0c56863e3&source=prChecksComment) |

:computer: Catch issues earlier using the plugins for [VS Code](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner), [JetBrains IDEs](https://plugins.jetbrains.com/plugin/10972-snyk-security), [Visual Studio](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner-vs-2022), and [Eclipse](https://marketplace.eclipse.org/content/snyk-security#external-install-button).

parfeon committed:
test(fixtures): fix fixtures after refactoring

chore(CODEOWNERS): add new SDK maintainer

Add `@mohitpubnub` as Ruby SDK maintainer.

mohitpubnub commented on lib/pubnub/events/here_now.rb:
```diff
@@ -52,11 +52,7 @@ def parameters(*_args)
-                 else
-                   [Pubnub::Constants::MAXIMUM_HERE_NOW_COUNT, @limit].min
-                 end
+        @limit = Pubnub::Constants::MAXIMUM_HERE_NOW_COUNT if @limit.nil?
```
So are we going to allow value more than `MAXIMUM_HERE_NOW_COUNT`?

mohitpubnub replied:
n/m


>limit higher than 1000 and negative offset are handled on server side.

I read this now

parfeon commented:
@pubnub-release-bot release

pubnub-release-bot committed:
PubNub SDK v6.0.1 release.
